plugin.js

```diff
@@ -57,30 +57,36 @@ function plugin (fastify, options, next) {
   const enableRotation = Array.isArray(secret)
   const signer = typeof secret === 'string' || enableRotation ? signerFactory(secret) : secret
 
-  fastify.decorate('parseCookie', function parseCookie (cookieHeader) {
+  fastify.decorate('parseCookie', parseCookie)
+  fastify.decorate('unsignCookie', unsginCookie)
```

Comment:
typo `unsignCookie`

Reply:
Thanks for letting me know, I'm not sure how my spellchecker didn't catch that.